android/src/main/java/com/dylanvann/fastimage/FastImageGlideUrlWithoutQueryParams.java

```diff
@@ -0,0 +1,25 @@
+package com.dylanvann.fastimage;
+
+import com.bumptech.glide.load.model.GlideUrl;
+import com.bumptech.glide.load.model.Headers;
+
+import java.net.URI;
+
+public class FastImageGlideUrlWithoutQueryParams extends GlideUrl {
+    private String staticURL;
+
+    public FastImageGlideUrlWithoutQueryParams(String url, Headers headers) {
+        super(url, headers);
+        staticURL = url.substring(0, url.lastIndexOf('?'));
```

Comment:
1) It might crash if url doesn't have ?
2) if url happen to have multiple ? it might incorrectly trim url (Shouldn't use indexOf() instead?)